cpp/models/ide_secir/parameters.h

```diff
@@ -47,6 +47,10 @@ namespace isecir
 /**
  * @brief Transition distribution for each transition in #InfectionTransition.
  *
+ * For each transition, the corresponding transition distribution can be chosen independently. 
+ * The choice of distributions determines how many initial time points are required to initialize the model, see
+ * get_global_support_max() in models/ide_secir/model.h.
```

Comment:
If the documentation of `get_global_support_max` redirects somewhere else again, this is going to be a wild goose chase of information hunting ;)